overlord/devicestate/firstboot.go

```diff
@@ -415,10 +444,13 @@ var loadDeviceSeed = func(st *state.State, sysLabel string) (deviceSeed seed.See
 		return assertstate.AddBatch(st, batch, nil)
 	}
 
-	if err := deviceSeed.LoadAssertions(assertstate.DB(st), commitTo); err != nil {
+	db := assertstate.DB(st)
+	if err := deviceSeed.LoadAssertions(db, commitTo); err != nil {
 		return nil, err
 	}
 
+	processAutoImportAssertions(st, deviceSeed, db, commitTo)
```

Comment:
this is probably not the the ideal place because loadDeviceSeed can end up being called multiple times. This belongs more in or after importAssertionsFromSeed

Reply:
This has been now moved after `importAssertionsFromSeed` call.